list.json

```diff
@@ -632,6 +632,12 @@
     "github": "samstefan/genesis"
   },
   {
+    "name": "Gerablog",
+    "github": "paulohrpinheiro/gerablog",
+    "license": "MIT",
+    "description": "Gerablog is a static blog generator. Write your posts in in *Markdown*, publish in *HTML*. Just two commands (create and generate) and seven templates (header, footer, categories page, categories block, main index, post and feed)."
+  },
+  {
```

Comment:
Oops, I'm sorry. I was looking for if there was any "is" with "static generator". I must have pressed space or something. It was not intentional, again, I'm sorry. I will fix.